lib/class-fm-demo-colorpicker.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ *
+ */
+
+if ( !class_exists( 'FM_Demo_Colorpicker' ) ) :
```

Comment:
WordPress coding standards reference seems to favor a space after `!`

`if ( ! class_exists( 'FM_Demo_Colorpicker' ) ) :`